lxd/instance/drivers/driver_qemu.go

```diff
@@ -2556,13 +2558,19 @@ chown -R root:root "${PREFIX}"
 		return err
 	}
 
-	// Udev rules
 	err = os.MkdirAll(filepath.Join(configDrivePath, "udev"), 0500)
 	if err != nil {
 		return err
 	}
 
-	lxdAgentRules := `ACTION=="add", SYMLINK=="virtio-ports/org.linuxcontainers.lxd", TAG+="systemd", ACTION=="add", RUN+="/bin/systemctl start lxd-agent.service"`
+	// Udev rules that trigger the start of the lxd-agent.service when qemu serail devices (symlinks
+	// in virtio-ports) appear.
```

Comment:
```suggestion
	// udev rules to start the lxd-agent.service when QEMU serial devices (symlinks in virtio-ports) appear.
```

Reply:
Actually, would it make sense to have the comment in the udev rule file itself?

Reply:
Yes, I'll add them into configuration within lxd-agent-loader package.